src/App.jsx

```diff
@@ -1,107 +1,28 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getCommentId(postId) {
+  return commentsFromServer.filter(comment => postId === comment.postId);
+}
+
+function getUserId(userId) {
+  return usersFromServer.find(user => userId === user.id)
+    || null;
```

Comment:
no needs to set `null` since you'll get false value anyway
```suggestion
  return usersFromServer.find(user => userId === user.id);
```